daisy_workflows/image_import/inflate_file.wf.json

```diff
@@ -38,6 +38,10 @@
     "import_license": {
       "Value": "projects/compute-image-tools/global/licenses/virtual-disk-import",
       "Description": "Import License used for tracking migration workflow use."
+    },
+    "compute_service_account": {
+      "Value": "default",
+      "Description": "Compute service account to run as"
```

Comment:
Similar to the comment for translation workflows, let's specify which instance the service account will be use diwth.